pagopa/pn-frontend
# fix: fixed inline link of digital contacts code verification modal

## Short description
Fixed inline link of digital contacts code verification modal

## List of changes proposed in this pull request
- Replaced ButtonNaked with Box in order to apply display inline 

## How to test
Go to "Recapiti" Page of PF and PG and try to insert an email or phone number. On the modal you should see the link "generane uno nuovo" inline
